### PR TITLE
Rely on the test config to set tests to skip

### PR DESCRIPTION
**Pull request by yujuhong**
Stop using a custom list of excluded tests

### CODE REVIEW

**Comment by yujuhong** [comment]
Going to merge this once https://github.com/kubernetes/test-infra/pull/11375 is in.

File: run-e2e.sh [hunk 1]
```diff
@@ -32,23 +32,11 @@ curl \
...
-curl https://raw.githubusercontent.com/e2e-win/e2e-win-prow-deployment/master/exclude_conformance_test.txt -o ${WORKSPACE}/exclude_conformance_test.txt
-# Ignore lines starting with "#"
-EXCLUDED_TESTS=$(cat ${WORKSPACE}/exclude_conformance_test.txt |
-  tr -d '\r' |                # remove Windows carriage returns
-  grep -v '^#' |              # ignore comment lines
-  grep -v '^$' |              # ignore blank lines
-  tr -s '\n' '|' |            # coalesce newlines into |
-  tr -s ' ' '.' |             # coalesce spaces into .
-  sed -e 's/[]\[()]/\\&/g' |  # escape brackets and parentheses
-  sed -e 's/.$//g')           # remove final | added by tr
```
Comment (pjh) [inline run-e2e.sh:45]: But I worked so hard on this command!

**Comment by pjh** [comment]
Woohoo!

File: run-e2e.sh (hunk 1 above)
Comment (yujuhong) [reply run-e2e.sh:45]: Haha It will forever live in the git history!

**Pull request merged**